src/main/java/hudson/plugins/jira/JiraIssueUpdater.java

```diff
@@ -48,6 +55,12 @@ public DescriptorImpl getDescriptor() {
         return DESCRIPTOR;
     }
 
+    public UpdaterIssuesSelector getIssueSelector() {
```

Comment:
Please conform to one naming standard i.e. using singular or plurar form for classes and methods.